infrastructure/terraform/modules/keyvault/kv.tf

```diff
@@ -15,17 +15,17 @@ resource "azurerm_key_vault" "key_vault" {
     virtual_network_subnet_ids = []
   }
   public_network_access_enabled = true
-  purge_protection_enabled      = true
```

Comment:
Why are we removing this and change the provider config?
Is this solving anything?